internal/frontend/templates/lobby.html

```diff
@@ -1150,6 +1180,53 @@
             }
         };
 
+        function handleGif(startNextGif) {
+            if (gifEnabled) {
+                if (gifLoading !== null) {
+                    gifLoading.then(gif => {
+                        clearInterval(frameIntervalId);
+                        const canvasData = context.getImageData(0, 0, context.canvas.width, context.canvas.height);
+                        gif.addFrame(canvasData, {delay: 100});
+                        gif.render();
+                    });
+                }
+
+                if (startNextGif) {
+                    let optionText = drawerName + ': Round ' + round.toString();
+                    gifLoading = fetch('https://cdn.jsdelivr.net/npm/gif.js@0.2.0/dist/gif.worker.js')
+                        .then((response) => {
+                            if (!response.ok)
+                                throw new Error("Network response was not OK");
+                            return response.blob();
+                        }).then(workerBlob => {
+                            let gif = new GIF({
+                                workers: 1,
+                                workerScript: URL.createObjectURL(workerBlob),
+                                quality: 10,
+                                width: context.clientWidth,
+                                height: context.clientHeight});
+
+                            gif.on('finished', function(blob) {
+                                const option = document.createElement('option');
+                                option.value = URL.createObjectURL(blob);
+                                option.innerHTML = optionText;
+                                gifUrlSelect.appendChild(option);
+                            });
+
+                            return gif;
+                        });
+
+                    gifLoading.then(gif => {
+                        const addFrame = function () {
+                            const canvasData = context.getImageData(0, 0, context.canvas.width, context.canvas.height);
+                            gif.addFrame(canvasData, {delay: 100});
+                        };
+                        frameIntervalId = setInterval(addFrame, 1000);
+                    });
+                }
+            }
+        }
```

Comment:
Could be nice to allow users to customize a bunch of this someday maybe - how often to add a frame, quality, how many workers. Also might like to add a final frame with the word in it. And maybe all the guesses as well.